Packs/CommonScripts/ReleaseNotes/1_4_24.md

```diff
@@ -1,4 +0,0 @@
-#### Scripts
```

Comment:
Don't remove this, as it's still a valid release note.

Reply:
unintentional, restoring.